controller/controller.go

```diff
@@ -255,6 +255,7 @@ func appHandler(c handlerConfig) http.Handler {
 	httpRouter.DELETE("/apps/:apps_id/formations/:releases_id", httphelper.WrapHandler(api.appLookup(api.DeleteFormation)))
 	httpRouter.GET("/apps/:apps_id/formations", httphelper.WrapHandler(api.appLookup(api.ListFormations)))
 	httpRouter.GET("/formations", httphelper.WrapHandler(api.GetFormations))
+	httpRouter.GET("/formations/active", httphelper.WrapHandler(api.ListActiveFormations))
```

Comment:
What do you think about making this a URL query parameter like `/formations?active=true` or `/formations?status=active`?